app/controllers/application_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
+
+  #helper_method :current_user
+   def current_user
+      User.find_by(id: session[:user_id])
+    end
+
+    def require_login
+      redirect_to root_path unless is_authenticated?
+    end
+
+    def is_authenticated?
+      !!session[:user_id]
+    end
```

Comment:
!!current_user might be better - what if someone logged (a vandal) then your admin people deleted their record to kick them out.